docs/pages/en-us/Architecture.md

```diff
@@ -13,26 +13,48 @@ banner:
 
 # Architecture
 
-The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch.
+The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch directly or via the Audit Web Service HTTP REST API.
 
-In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch.
+In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch or the Audit Web Service HTTP REST API.
 
-Elasticsearch receives CAF audit events for a tenant from the client-side library and adds the application audit event to the tenant's index.
+Elasticsearch receives application audit events for a tenant and stores application audit events in the tenant's index.
 
 ## Overview
 
-Auditing is built on Elasticsearch for the messaging and storage of the audit events. Elasticsearch offers high availability, throughput, scalability, and performance to the overall solution. Additionally, Elasticsearch is accessible via RESTful APIs, it offers strong data analytics and monitoring capabilities.
+Auditing is built on Elasticsearch for the messaging and storage of audit events. Elasticsearch offers high availability, throughput, scalability, and performance to the overall solution. Additionally, Elasticsearch is accessible via RESTful APIs, it offers strong data analytics and monitoring capabilities.
+
+The Audit Service client-side library can be configured to send application audit event messages directly to Elasticsearch or via the Audit Web Service HTTP REST API.
 
 ## Audit Service Component Architecture
```

Comment:
In the latest commit:
- I am now illustrating the solution with a single diagram.
- I have removed the configuration paragraph from the Overview. I have then moved the Overview heading to sit below the Architecture heading and moved the, "Auditing is built on Elasticsearch for the messaging and...", paragraph to a more appropriate position.